src/Carnac.Logic/MessageController.cs

```diff
@@ -60,49 +66,29 @@ public void Start()
                         .Switch()
                         .Select(_ => message)
                         .Take(1);
-                })
-                .Select(m => Tuple.Create(m, ActionType.FadeOut));
+                });
+            var fadeOutMessageSubscription = fadeOutMessageStream
+                .ObserveOn(concurrencyService.MainThreadScheduler)
+                .SubscribeOn(concurrencyService.MainThreadScheduler)
+                .Subscribe(m => m.IsDeleting = true);
```

Comment:
Yep, there is an issue open. Unsure of what should happen when there is an error, there never has been any error handling and need to figure out strategy first